Postgres/AppDelegate.m

```diff
@@ -183,6 +205,14 @@ - (NSApplicationTerminateReply)applicationShouldTerminate:(NSApplication *)sende
     return NSTerminateLater;
 }
 
+- (BOOL)isDarkMode {
+    return [[[NSUserDefaults standardUserDefaults] valueForKey:kAppleInterfaceStyle] isEqualToString:kAppleInterfaceStyleDark];
+}
+
+- (void)dealloc {
+    [[NSDistributedNotificationCenter defaultCenter] removeObserver:self];
```

Comment:
Instead of `self` you have to pass the observer object returned by `addObserverForName:object:queue:usingBlock:`.